spec/ParseQuery.spec.js

```diff
@@ -2860,31 +2860,17 @@ describe('Parse.Query testing', () => {
     });
   });
 
-  it('object with length', function (done) {
+  it('object with length', async () => {
```

Comment:
Do we need an extra test case for each field name, or does it makes sense to consolidate into a single test case with a loop over an array of field names?

Reply:
Done